personal_notion_bot/telegram_side/loader.py

```diff
@@ -0,0 +1,11 @@
+import sys; sys.path.append("..")
+from aiogram import Bot, Dispatcher, types, executor
+from aiogram.contrib.fsm_storage.memory import MemoryStorage
+
+from config import BOT_TOKEN
+
+
+bot = Bot(token = BOT_TOKEN)
+storage = MemoryStorage()
+dp = Dispatcher(bot, storage=storage)
```

Comment:
It is a good practice to keep names as clear as possible. In this case, it would be better to name those variables as 
```suggestion
bot = Bot(token = BOT_TOKEN)
memory_storage = MemoryStorage()
dispatcher = Dispatcher(bot, storage=storage)
```

In this manner it is clear from the beginning what the variables represent for other code readers.